### PR TITLE
[Docs] Pick seg fastdeploy docs from PaddleSeg

### PR DESCRIPTION
**Pull request by DefTruth**
### PR types(PR类型)

Doc  

### Description

- 将PaddleSeg/deploy/fastdeploy更新到FastDeploy中的seg examples

### CODE REVIEW

**Comment by rainyfly** [comment]
近期paddleseg下增加了C和C#的使用示例，里面的README文件里的问档链接可能需要跟随目录的改动而改动
examples/vision/segmentation/paddleseg/cpu-gpu/c/README.md
examples/vision/segmentation/paddleseg/cpu-gpu/c/README_CN.md
examples/vision/segmentation/paddleseg/cpu-gpu/csharp/README.md
examples/vision/segmentation/paddleseg/cpu-gpu/csharp/README_CN.md

File: examples/vision/segmentation/paddleseg/semantic_segmentation/cpu-gpu/c/README.md [hunk 1]
```diff
@@ -180,5 +180,5 @@ FD_C_Mat FD_C_VisSegmentation(FD_C_Mat im,
 
 - [PPSegmentation Model Description](../../)
 - [PaddleSeg Python Deployment](../python)
-- [Model Prediction Results](../../../../../../docs/api/vision_results/)
-- [How to switch the model inference backend engine](../../../../../../docs/cn/faq/how_to_change_backend.md)
+- [Model Prediction Results](https://github.com/PaddlePaddle/FastDeploy/blob/develop/docs/api/vision_results/)
+- [How to switch the model inference backend engine](https://github.com/PaddlePaddle/FastDeploy/blob/develop/docs/cn/faq/how_to_change_backend.md)
```
Comment (rainyfly) [inline examples/vision/segmentation/paddleseg/semantic_segmentation/cpu-gpu/c/README.md]: https://github.com/PaddlePaddle/FastDeploy/blob/develop/docs/en/faq/how_to_change_backend.md
不好意思，这个我之前写错了

**Review by rainyfly — APPROVED** [review]
LGTM for c and c# directory movement

File: examples/vision/segmentation/paddleseg/semantic_segmentation/cpu-gpu/c/README.md (hunk 1 above)
Comment (DefTruth) [reply examples/vision/segmentation/paddleseg/semantic_segmentation/cpu-gpu/c/README.md]: done

**Pull request merged**